src/modules/esl-carousel/core/esl-carousel.ts

```diff
@@ -306,6 +306,15 @@ export class ESLCarousel extends ESLBaseElement {
     return el && el.hasAttribute('prev');
   }
 
+  // Navigates to requested slide
+  @listen('esl:show:request')
+  protected onHandleFootnotes(e: CustomEvent): void {
+    const target = (e.target as HTMLElement).closest('[esl-carousel-slide]') as HTMLElement;
```

Comment:
It looks good, but not perfect, just a couple of things we should consider here:

1) We have a really rare but realistic case when we have a nested carousel.
So in that case we will navigate trough the child carousel but we will not affect top parent carousel as the  `(e.target as HTMLElement).closest('[esl-carousel-slide]')` will give us only the most nested slide which is not member of parent carousel slide list.
We can discuss a solution for that case within the bounds of the current PR, or create that as a separate issue (If we really should harry up).

2) We had the support of the ` if (!isMatches(this, detail.match)) return null;` filter in toggleable, so we need to be consistent here as well.